gapis/api/gles/compat.go

```diff
@@ -562,6 +562,13 @@ func compat(ctx context.Context, device *device.Instance) (transform.Transformer
 				log.E(ctx, "Error decompressing texture: %v", err)
 			}
 
+		case *GlTexBufferEXT:
+			if !version.IsES { // Strip suffix on desktop.
```

Comment:
You know that the main reason that the recent APK was crashing was proliferation of boolean flags though compat, right?